src/blockdata/opcodes.rs

```diff
@@ -651,7 +651,7 @@ impl fmt::Debug for All {
             all::OP_CHECKMULTISIGVERIFY => write!(f, "CHECKMULTISIGVERIFY"),
             all::OP_CLTV => write!(f, "CLTV"),
             all::OP_CSV => write!(f, "CSV"),
-            All {code: x} if x >= all::OP_NOP1.code && x <= all::OP_NOP10.code => write!(f, "NOP{}", x - all::OP_NOP1.code + 1),
+            All {code: x} if (all::OP_NOP1.code..=all::OP_NOP10.code).contains(&x) => write!(f, "NOP{}", x - all::OP_NOP1.code + 1),
```

Comment:
I am still getting used to the new syntax here. But gotta learn sometime

Reply:
I didn't even like it at first but got convinced by argument that it prevents messing up of operators and such.